### PR TITLE
[Outlook](Script Lab for Outlook) Updates current SL article to include Outlook

### PR DESCRIPTION
**Pull request by mscharlock**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[a633763](https://github.com/OfficeDev/office-js-docs-pr/commits/a633763cc0a606a4a6218ef67e6531f22265c792)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/overview/explore-with-script-lab.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-sl4o/docs/overview/explore-with-script-lab.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/overview/explore-with-script-lab?branch=pr-en-us-1951) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C6%5C11%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202006110531128962-1951%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=yRF6LlFMfLfWXrSkJOHroRwjyJ%2BVU8Jf00iewgUjzZQ%3D&st=2020-06-11T05%3A27%3A41Z&se=2020-07-12T05%3A32%3A41Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/overview/explore-with-script-lab.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+ms.date: 06/10/2020
 ms.topic: conceptual
 ms.custom: scenarios:getting-started
 localization_priority: Priority
 ---
 
 # Explore Office JavaScript API using Script Lab
 
-The [Script Lab add-in](https://appsource.microsoft.com/product/office/WA104380862), which is available free from AppSource, enables you to explore the Office JavaScript API while you're working in an Office program such as Excel or Word. Script Lab is a convenient tool to add to your development toolkit as you prototype and verify functionality you want in your add-in.
+The [Script Lab add-in](https://appsource.microsoft.com/product/office/WA104380862) and [Script Lab for Outlook add-in](https://appsource.microsoft.com/en-us/product/office/WA200001603?tab=Overview), which are available free from AppSource, enable you to explore the Office JavaScript API while you're working in an Office program such as Excel or Word. Script Lab is a convenient tool to add to your development toolkit as you prototype and verify functionality you want in your add-in.
```
Comment (ElizabethSamuel-MSFT) [inline docs/overview/explore-with-script-lab.md:12]:
```suggestion
The [Script Lab](https://appsource.microsoft.com/product/office/WA104380862) and [Script Lab for Outlook](https://appsource.microsoft.com/product/office/WA200001603) add-ins, available free from AppSource, enable you to explore the Office JavaScript API while you're working in an Office program such as Excel or Outlook. Script Lab is a convenient tool to add to your development toolkit as you prototype and verify functionality you want in your own add-in.
```

File: docs/overview/explore-with-script-lab.md [hunk 2]
```diff
@@ -56,10 +56,20 @@ Script Lab is supported for Excel, Word, and PowerPoint on the following clients
...
+- Office 2016 or later on Mac
+- Office on the web when using Chrome, Microsoft Edge, or Safari browsers.
+
+For complete details on Script Lab for Outlook, see the [Script Lab now supports Outlook blog post](https://developer.microsoft.com/en-us/office/blogs/script-lab-now-supports-outlook/) available on the Office 365: Developer Blog.
+
 ## Next steps
 
 To use Script Lab in Excel, Word, or PowerPoint, install the [Script Lab add-in](https://appsource.microsoft.com/product/office/WA104380862) from AppSource. 
 
+To use Script Lab for Outlook, install the [Script Lab for Outlook add-in](https://appsource.microsoft.com/en-US/product/office/wa200001603?tab=overview) from AppSource.
```
Comment (ElizabethSamuel-MSFT) [inline docs/overview/explore-with-script-lab.md:71]:
```suggestion
To use Script Lab for Outlook, install the [Script Lab for Outlook add-in](https://appsource.microsoft.com/product/office/wa200001603) from AppSource.
```

File: docs/overview/explore-with-script-lab.md [hunk 3]
```diff
@@ -56,10 +56,20 @@ Script Lab is supported for Excel, Word, and PowerPoint on the following clients
 - Office 2016 or later on Mac
 - Office on the web
 
+Script Lab for Outlook is available on the following clients.
+
+- Outlook 2013 or later on Windows
+- Office 2016 or later on Mac
+- Office on the web when using Chrome, Microsoft Edge, or Safari browsers.
+
+For complete details on Script Lab for Outlook, see the [Script Lab now supports Outlook blog post](https://developer.microsoft.com/en-us/office/blogs/script-lab-now-supports-outlook/) available on the Office 365: Developer Blog.
```
Comment (ElizabethSamuel-MSFT) [inline docs/overview/explore-with-script-lab.md:65]:
```suggestion
For more details on Script Lab for Outlook, see the related [blog post](https://developer.microsoft.com/outlook/blogs/script-lab-now-supports-outlook/).
```

File: docs/overview/explore-with-script-lab.md [hunk 4]
```diff
@@ -56,10 +56,20 @@ Script Lab is supported for Excel, Word, and PowerPoint on the following clients
 - Office 2016 or later on Mac
 - Office on the web
 
+Script Lab for Outlook is available on the following clients.
+
+- Outlook 2013 or later on Windows
+- Office 2016 or later on Mac
```
Comment (ElizabethSamuel-MSFT) [inline docs/overview/explore-with-script-lab.md:62]:
```suggestion
- Outlook 2016 or later on Mac
```

File: docs/overview/explore-with-script-lab.md [hunk 5]
```diff
@@ -56,10 +56,20 @@ Script Lab is supported for Excel, Word, and PowerPoint on the following clients
...
+- Office on the web when using Chrome, Microsoft Edge, or Safari browsers.
```
Comment (ElizabethSamuel-MSFT) [inline docs/overview/explore-with-script-lab.md:63]:
```suggestion
- Outlook on the web when using Chrome, Microsoft Edge, or Safari browsers
```

File: docs/overview/explore-with-script-lab.md [hunk 6]
```diff
@@ -56,10 +56,20 @@ Script Lab is supported for Excel, Word, and PowerPoint on the following clients
...
+
 ## Next steps
 
 To use Script Lab in Excel, Word, or PowerPoint, install the [Script Lab add-in](https://appsource.microsoft.com/product/office/WA104380862) from AppSource. 
 
+To use Script Lab for Outlook, install the [Script Lab for Outlook add-in](https://appsource.microsoft.com/en-US/product/office/wa200001603?tab=overview) from AppSource.
+
 You're welcome to expand the sample library in Script Lab by contributing new snippets to the [office-js-snippets](https://github.com/OfficeDev/office-js-snippets#office-js-snippets) GitHub repository.
 
 When you're ready to create your first Office Add-in, try out the quick start for [Excel](../quickstarts/excel-quickstart-jquery.md), [Outlook](../quickstarts/outlook-quickstart.md), [Word](../quickstarts/word-quickstart.md), [OneNote](../quickstarts/onenote-quickstart.md), [PowerPoint](../quickstarts/powerpoint-quickstart.md), or [Project](../quickstarts/project-quickstart.md).
```
Comment (ElizabethSamuel-MSFT) [inline docs/overview/explore-with-script-lab.md:75]: In the See also section, there's a link to "Get Script Lab". We'll need to add an entry to "Get Script Lab for Outlook".

**Comment by o365devx** [comment]
Docs Build status updates of commit _[812daaf](https://github.com/OfficeDev/office-js-docs-pr/commits/812daafbac2b22333acc614f9feb6b3d40217670)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/overview/explore-with-script-lab.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-sl4o/docs/overview/explore-with-script-lab.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/overview/explore-with-script-lab?branch=pr-en-us-1951) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C6%5C11%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202006111901084497-1951%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=R%2FgIyVYk8TU24FEt0zrkc83UNawm7u5FOvKrW6v8Fpc%3D&st=2020-06-11T18%3A57%3A41Z&se=2020-07-12T19%3A02%3A41Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[3f701a9](https://github.com/OfficeDev/office-js-docs-pr/commits/3f701a9c1772664e5a2764b3b8608d6467da8d6c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/overview/explore-with-script-lab.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-sl4o/docs/overview/explore-with-script-lab.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/overview/explore-with-script-lab?branch=pr-en-us-1951) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C6%5C11%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202006111902108715-1951%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=AzH7MzGuK5st%2BhcUAEZ6hiL5e4AmdV3saWl5vIAiXXs%3D&st=2020-06-11T18%3A59%3A17Z&se=2020-07-12T19%3A04%3A17Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/overview/explore-with-script-lab.md [hunk 7]
```diff
@@ -1,19 +1,19 @@
...
 
 ## What is Script Lab?
 
-Script Lab is a tool for anyone who wants to learn how to develop Office Add-ins using the Office JavaScript API in Excel, Word, or PowerPoint. It provides IntelliSense so you can see what's available and is built on the Monaco framework, the same framework used by Visual Studio Code. Through Script Lab, you can access a library of samples to quickly try out features or you can use a sample as the starting point for your own code. You can even use Script Lab to try preview APIs.
+Script Lab is a tool for anyone who wants to learn how to develop Office Add-ins using the Office JavaScript API in Excel, Word, PowerPoint, and Outlook. It provides IntelliSense so you can see what's available and is built on the Monaco framework, the same framework used by Visual Studio Code. Through Script Lab, you can access a library of samples to quickly try out features or you can use a sample as the starting point for your own code. You can even use Script Lab to try preview APIs.
```
Comment (ElizabethSamuel-MSFT) [inline docs/overview/explore-with-script-lab.md:16]:
```suggestion
Script Lab is a tool for anyone who wants to learn how to develop Office Add-ins using the Office JavaScript API in Excel, Outlook, Word, and PowerPoint. It provides IntelliSense so you can see what's available and is built on the Monaco framework, the same framework used by Visual Studio Code. Through Script Lab, you can access a library of samples to quickly try out features or you can use a sample as the starting point for your own code. You can even use Script Lab to try preview APIs.
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[7698a62](https://github.com/OfficeDev/office-js-docs-pr/commits/7698a628d0e76c1e6748820427ece114a2c299b8)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/overview/explore-with-script-lab.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-sl4o/docs/overview/explore-with-script-lab.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/overview/explore-with-script-lab?branch=pr-en-us-1951) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C6%5C12%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202006121809172997-1951%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=TvJNdZV6AfKgjOfUbLU86MYfEjLs4%2F2C0AtRISjKmQo%3D&st=2020-06-12T18%3A05%3A41Z&se=2020-07-13T18%3A10%3A41Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[d57322d](https://github.com/OfficeDev/office-js-docs-pr/commits/d57322d9cf62e8fa500e67d658d604ee5015b5fa)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/overview/explore-with-script-lab.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-sl4o/docs/overview/explore-with-script-lab.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/overview/explore-with-script-lab?branch=pr-en-us-1951) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C6%5C12%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202006121811380840-1951%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=qsh5VyMnNb%2BXHmlJz8KNOHMCk31yvPE%2BKsleZNa%2FfL8%3D&st=2020-06-12T18%3A08%3A04Z&se=2020-07-13T18%3A13%3A04Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Pull request merged**